tools/rostopic/src/rostopic/__init__.py

```diff
@@ -100,7 +100,7 @@ class ROSTopicHz(object):
     def __init__(self, window_size, filter_expr=None):
         import threading
         self.lock = threading.Lock()
-        self.last_printed_tn = 0
+        self.last_msg_tn = 0
```

Comment:
Please do not change the name of this variable. It is publically visible and it would break the API to rename it.